File: C/PRAK405-2210817210005-MFithra_Siroja.c
#include<stdio.h>

int main(void){
int a,b,c,d,hasil,total;
scanf("%d %d", &a, &b);
for (c=1;c<=a;c++){
     for(d=c;d>0;d--){printf("(%d * %d)", d,b);
         if(d>1){
            printf(" + ");}}
    hasil += (c)*b;
    printf(" = %d\n", hasil);
    total += hasil;
    }
printf("%d", total);
}

File: Python/PRAK405-2210817210005-MFithra_Siroja.py
n1,n2= map( int,input().split())
hasil=0;jumlah=0;total=0
a=0
while a<n1:
    a += 1
    b=a
    if a == 0:
        break
    while b>0:
        b -= 1
        print((b+1, n2), end= ' ')
        if b>0 :
            print(' + ', end=' ')
    hasil = a*n2
    jumlah += hasil
    print(jumlah)
    total += jumlah
print(total)